x-pack/solutions/security/plugins/security_solution/public/overview/components/detection_response/utils.tsx

```diff
@@ -5,13 +5,24 @@
  * 2.0.
  */
 
+import type { EuiThemeComputed } from '@elastic/eui';
+
+// @deprecated, use getSeverityColor instead
 export const SEVERITY_COLOR = {
   critical: '#E7664C',
   high: '#DA8B45',
   medium: '#D6BF57',
   low: '#54B399',
 } as const;
 
+export const getSeverityColor = (euiTheme: EuiThemeComputed) =>
+  ({
+    critical: euiTheme.colors.vis.euiColorVis9,
```

Comment:
The Borealis data viz color palette changed, see https://docs.google.com/document/d/1IAKbasq1nDfqd2IU3KdP8cwD3uCCAwkIekKRq7zgyWg/edit?tab=t.0#heading=h.lgyarxm75dlu , so that vis9 is not red but yellow and vis5 is not yellow but pink. These colors don't work for severity
There is Severity Palette introduced by the EUI team https://github.com/elastic/kibana/issues/203387?reload=1 , but it is missing green color, which is currently how our low severity alerts/vulnerabilities are being show. There is a discussion with Security solution on how to go about it, see https://elastic.slack.com/archives/C060SDYTZ96/p1734636026436679

Reply:
Thanks @maxcold , I could find that we are adding more `euiColorSeverity` color for different severity levels. This PR might have to wait until those colors becomes available. Have we got a clear idea of which `euiColorSeverity` token should be used for critical, high, medium and low?

Reply:
@angorayc the team is moving from the variety of colors illustrated by your screenshot to a unified palette available here https://github.com/elastic/eui/pull/8247 , but I haven't seen exact colors defined for Security severity levels from this palette. If I understand correctly, it's what @codearos is doing currently with the design team